sharding-ui/sharding-ui-frontend/src/components/Logo/index.vue

```diff
@@ -18,8 +18,8 @@
 <template>
   <div class="s-pro-components-sider-menu-index-logo">
     <a href="/">
-      <img class src="../../assets/img/sidebar-logo.png" alt="logo">
-      <img class="collapse-logo" src="../../assets/img/logo.png" alt="logo">
+      <img src="../../assets/img/sidebar-logo.png" alt="logo" />
```

Comment:
```suggestion
      <img src="../../assets/img/sidebar-logo.png" alt="logo" >
```
Here is eslint error: Disallow self-closing on HTML void elements.

Reply:
add eslint rules，solve the problem of adding self-closing tags，the snippet code is as follows:

```
"vue/html-self-closing": ["error", {
      "html": {
        "void": "any",
        "normal": "any",
        "component": "any"
      },
      "svg": "any",
      "math": "any"
    }]

```